core/templates/pages/contributor-dashboard-page/translation-language-selector/translation-language-selector.component.ts

```diff
@@ -66,6 +67,11 @@ export class TranslationLanguageSelectorComponent implements OnInit {
       .then((featuredLanguages: FeaturedTranslationLanguage[]) => {
         this.featuredLanguages = featuredLanguages;
       });
+
+    this.activeLanguageCode ? (
+      this.languageSelection = this.languageIdToDescription[
+        this.activeLanguageCode]) : this.languageSelection = (
+          'Select a language...');
```

Comment:
Huh this syntax seems strange to me. Can't this be:

```suggestion 
      this.languageSelection = (
        this.activeLanguageCode ?
        this.languageIdToDescription[this.activeLanguageCode] : 
        'Select a language...'
      );
```

Reply:
Done